### DISCUSSION TITLE
clangd lsp not attaching after updating clangd_extensions.nvim

### DISCUSSION THREAD

**Discussion by moonstruckmoth** [LazyVim/LazyVim#1323] [Q&A] [answered]
Seems like there was a [breaking change](https://github.com/p00f/clangd_extensions.nvim/commit/643fd64f908eaa599469cb0af9a204a0dbd3b7e2), in terms of configuring clangd_extensions.
After this update, clangd lsp has stopped attaching.

Do we need to change anything on LazyVim's side?

**Comment by dpetka2001** [comment, marked as answer]
See #1308 .

**Reply by moonstruckmoth** [reply]
Thanks, it worked for me.
